library/std/src/sys/unix/time.rs

```diff
@@ -303,6 +303,7 @@ mod inner {
         pub fn actually_monotonic() -> bool {
             (cfg!(target_os = "linux") && cfg!(target_arch = "x86_64"))
                 || (cfg!(target_os = "linux") && cfg!(target_arch = "x86"))
+                || (cfg!(target_os = "linux") && cfg!(target_arch = "aarch64"))
```

Comment:
Are there any Linux targets left where the issue is not already fixed by the kernel?

Reply:
Linux does include a self-test `linux/tools/testing/selftests/timers/inconsistency-check` that checks for monotonicity, and the manpage for `clock_gettime()` says :
>CLOCK_MONOTONIC
              Clock that cannot be set and represents monotonic time since some unspecified starting point.  This clock is  not  affected  by discontinuous  jumps  in the system time (e.g., if the system administrator manually changes the clock), but is affected by the incremental adjustments performed by adjtime(3) and NTP.

So it seems like Linux is making a user-api guarantee here, but i can't seem to find anything explicitly saying that.